File: app/src/main/java/io/github/pierry/better_call_me/domain/viewmodels/MoreInfo.java
package io.github.pierry.better_call_me.domain.viewmodels;

import com.google.gson.annotations.SerializedName;

public class MoreInfo {

  private Period month;
  private Period year;
  @SerializedName("12months") private Period twelveMonths;

  public Period getMonth() {
    return month;
  }

  public void setMonth(Period month) {
    this.month = month;
  }

  public Period getYear() {
    return year;
  }

  public void setYear(Period year) {
    this.year = year;
  }

  public Period getTwelveMonths() {
    return twelveMonths;
  }

  public void setTwelveMonths(Period twelveMonths) {
    this.twelveMonths = twelveMonths;
  }
}

File: app/src/main/java/andcordeiro/com/br/testeandroid/entities/MoreInfo.kt
package andcordeiro.com.br.testeandroid.entities

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class MoreInfo: Serializable {

    @SerializedName("month")
    var month: Month? = null
    @SerializedName("year")
    var year: Year? = null
    @SerializedName("12months")
    var twelveMonths: TwelveMonths? = null

    override fun toString(): String {
        return "MoreInfo(month=$month, year=$year, twelveMonths=$twelveMonths)"
    }
}